modules/management/src/test/java/org/mule/management/config/ManagementNamespaceHandlerTestCase.java

```diff
@@ -107,18 +107,18 @@ public void testAgentsOrder() throws Exception
         assertEquals(agents.size(), 8);
         
         Iterator<Agent> iter = agents.iterator();
-        assertTrue(iter.next() instanceof JmxApplicationAgent);
```

Comment:
Don't understand this test: the assertion checking that JmxApplicationAgent was in the list was removed, but then there is a new assertion checking that it is present in the iterator.